### PR TITLE
Update README.MD

### PR DESCRIPTION
**Pull request by aftex261**
**Fixes issue:**#4478

**Changes:**

### CODE REVIEW

**Comment by aftex261** [comment]
> return type of functions should be kept in the separate line as done originally

I have done it now as you said

**Comment by aftex261** [comment]
Please check my issue .It's more than 4 days .please review and merge it.

File: guides/coding_style/c/README.md [hunk 1]
```diff
@@ -27,16 +27,25 @@ main(int argc, char *argv[])
...
+	{
 		some_other_thing();
-
+              }
 	return (0);
 }
 ```
 
+*argc stands for arguments count(ARGument count)
+*argv stands for arguments values(ARGument vector)
```
Comment (arnavb) [inline guides/coding_style/c/README.md]:
```suggestion
*argv stands for arguments values(ARGument values)
```

File: guides/coding_style/c/README.md [hunk 2]
```diff
@@ -27,16 +27,25 @@ main(int argc, char *argv[])
 	if (!some_function()) {
 		puts("Something happened!");
 		do_a_thing();
-	} else
+	} 
+	else
+	{
 		some_other_thing();
-
+              }
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: Don't modify the existing code for the if/else.

File: guides/coding_style/c/README.md [hunk 3]
```diff
@@ -27,16 +27,25 @@ main(int argc, char *argv[])
...
+*argc stands for arguments count(ARGument count)
+*argv stands for arguments values(ARGument vector)
+*argv[0] is the name of the program 
+
+
 ## Indentation
 
 Indentation is done with a single tab character (Hard Tab). For code split across multiple lines a helper indent of 4 spaces is used.
+In C Indentation is not must .It happens on default by the compiler.
+But if you are on python then Indentation is must.
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: Delete these two lines.

File: guides/coding_style/c/README.md [hunk 4]
```diff
@@ -48,7 +57,8 @@ some_really_long_function(int a, int b, int c, int d,
 
 ## Conditionals
 
-If, else, for, while, and switch statements should be followed by a space.
+If statement, else statement, for loop, while loop, and switch statements should be followed by a space.
+Inside these conditional statements if expresssion value is positive or negative then it evaluates to true but if it is true then it evaluates to false
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: Again, this is useless information for a style guide.

File: guides/coding_style/c/README.md [hunk 5]
```diff
@@ -58,7 +68,10 @@ if (a == 9) {
 for (;;)
 ```
 
-Each case in a switch statement should not be indented but the code for each should be. Any case falthroughs should be commented.
+Each case in a switch statement should not be indented but the code for each should be. Any case fallthroughs should be commented.
+Every case should be terminated with a break statement.
+Default statement is not compulsory to be used.
+Inside switch case statement continue can't be used.Continue statement is used inside loops only.
```
Comment (arnavb) [inline guides/coding_style/c/README.md]:
```suggestion
Inside switch case statement continue can't be used. Continue statement is used inside loops only.
```

File: guides/coding_style/c/README.md [hunk 6]
```diff
@@ -84,21 +97,29 @@ Functions should have the type on a seperate line proceeding the rest of the fun
...
 main(int argc, char *argv[])
 {
+Block of Code
+" " "
+" " "
+}
 ```
 
-Return statements should have the value wrapped in parenthesis.
+Return statements should have the value wrapped in parenthesis.There are basically 4 types of jump statements:-
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: The information provided at the end of this is not needed.

File: guides/coding_style/c/README.md [hunk 7]
```diff
@@ -84,21 +97,29 @@ Functions should have the type on a seperate line proceeding the rest of the fun
 int
 main(int argc, char *argv[])
 {
+Block of Code
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: Precede with `//` comments for each line.

File: guides/coding_style/c/README.md [hunk 8]
```diff
@@ -84,21 +97,29 @@ Functions should have the type on a seperate line proceeding the rest of the fun
...
+*break
 
 ```C
 return (0);
 ```
 
 ## Comments
 
-Always use C style comments (`/* */`) and not C++ style comments (`//`). A sample is shown below.
+Always use C style comments (`/* */`) and C++ style comments (`//`). A sample is shown below.
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: not was correct. In C, many compilers (older ones) do not support singe line comments.

File: guides/coding_style/c/README.md [hunk 9]
```diff
@@ -84,21 +97,29 @@ Functions should have the type on a seperate line proceeding the rest of the fun
...
 
 ```C
 
-/* One line comment */
+/* One line comment */ or  //For one line comment
```
Comment (arnavb) [inline guides/coding_style/c/README.md]: See above.

**Comment by aftex261** [comment]
Sir, I have committed all the changes you said. Please kindly review and merge it.

**Pull request merged**